fib-prime.js

```diff
@@ -1,34 +1,35 @@
-var ispnum = function(num) {
-  for(var i = 2; i < num; i++)
-    if  (num % i === 0) return false;
-  return num > 1;
-};
+const isPrime = num => {
+  for (let i = 2, s = Math.sqrt(num); i <= s; i++)
```

Comment:
Used Euler simple logic to reduce the complexity of the algorithm. 
O(n) => O(sqrt(n))